app/controllers/users/registrations_controller.rb

```diff
@@ -27,7 +27,7 @@ def validate_email_on_whitelist
 
   def set_user_object_with_errors
     @user = User.new(sign_up_params)
-    @user.errors.add(:email, 'must be from a government domain')
+    @user.errors.add(:email, "must be from a government domain. If you're having trouble signing up, <a href='/help/new'>contact us</a>.")
```

Comment:
is there a way to load the href from the routes, instead of it being hard coded?